emukit/bayesian_optimization/acquisitions/entropy_search.py

```diff
@@ -208,3 +215,70 @@ def _innovations(self, x: np.ndarray) -> tuple:
     def has_gradients(self) -> bool:
         """Returns that this acquisition has gradients"""
         return False
+
+
+class MultiInformationSourceEntropySearch(EntropySearch):
+    """
+    Entropy search acquisition for multi-information source problems where the objective function is the output of one
+    of the information sources. The other information sources provide auxiliary information about the objective function
+    """
+    def __init__(self, model: Union[IModel, IEntropySearchModel], space: ParameterSpace,
+                 target_information_source_index: int=None, num_samples: int = 100,
+                 num_representer_points: int = 50, burn_in_steps: int = 50):
+        """
+        :param model: Gaussian process model of the objective function that implements IEntropySearchModel
+        :param space: Parameter space of the input domain
+        :param target_information_source_index: The index of the information source we want to minimise
+        :param num_samples: Integer determining how many samples to draw for each candidate input
+        :param num_representer_points: Integer determining how many representer points to sample
+        :param burn_in_steps: Integer that defines the number of burn-in steps when sampling the representer points
+        """
+
+        # Find information source parameter in parameter space
+        info_source_parameter = None
+        self.fidelity_idx = None
```

Comment:
Is it right to call this fidelity if this class is general for muti output/fidelity cases?